extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/photos/ImgurPhotosExporter.java

```diff
@@ -54,14 +53,12 @@
 /** Exports Imgur albums and photos using Imgur API */
 public class ImgurPhotosExporter
     implements Exporter<TokensAndUrlAuthData, PhotosContainerResource> {
-  private static final String BASE_URL = ImgurTransferExtension.BASE_URL;
   private static final String RESULTS_PER_PAGE = "10";
-  private static final String ALBUM_PHOTOS_URL_TEMPLATE = BASE_URL + "/album/%s/images";
-  private static final String ALBUMS_URL_TEMPLATE =
-      BASE_URL + "/account/me/albums/%s?perPage=" + RESULTS_PER_PAGE;
-  private static final String ALL_PHOTOS_URL_TEMPLATE =
-      BASE_URL + "/account/me/images/%s?perPage=" + RESULTS_PER_PAGE;
-  private static final String DEFAULT_ALBUM_ID = "defaultAlbumId";
+  private static String albumPhotosUrlTemplate;
```

Comment:
Can we remove static and add final to these. That way they, after they are assigned in the constructor, the won't be modified.

Reply:
Done